src/util/MIDILoader.js

```diff
@@ -0,0 +1,18 @@
+const MIDILoader = (file, onLoad, dispatch) => {
+  try {
+    if (!['audio/midi', 'audio/x-midi', 'audio/mid'].includes(file.type)) {
+      dispatch({
+        type: 'SNACK_LOAD',
+        message: 'Invalid mime type',
+        variant: 'error'
+      })
```

Comment:
Throw `Error` here, and dispatch `SNACK_LOAD` in the `catch` clause.

Reply:
To fix the warning “'throw' of exception caught locally”, you should not use throw a new error to control flow in try-catch block. (maybe)
See https://stackoverflow.com/questions/47015693/how-to-fix-throw-of-exception-caught-locally

Reply:
Hmm. We should not ignore such warnings unless we have proper reason, and I feel we don't have it here. Maybe we should leave them untouched...

Reply:
:+1: